src/mame/drivers/at.cpp

```diff
@@ -1709,15 +1685,146 @@ ROM_START ( mkp286 )
 	ROMX_LOAD( "286-morse_kp286_hi.bin", 0x10001, 0x8000, CRC(a5f640e0) SHA1(7bbb7fce54079005cb691816d2301a3eda475a82), ROM_SKIP(1) )
 ROM_END
 
-// Motherboards using the 5 chip VLSI chipset
 
-// MG Products (Japanese) - Chipset: VLSI VL82C102A; VLSI VL82C101B; VLSI VL82C104; VLSI VL82C103A; VLSI VL82C100; (VLSI 8908BT; 8906BT; 8852BT; 8907BT; 8906BT)
-// BIOS: AMI 286 BIOS+ - BIOS-String: D286-9987-092588-K0 - ISA8: 2, ISA16: 8
-// ROM_SYSTEM_BIOS(27, "286vlsij", "Japanese 286 VLSI") /* (BIOS release date:: 25-09-1988) (ISA8: 3, ISA16: 5) */
+// WYSEpc 286 - motherboard: WY-2200-01 - continuous ticks from the speaker
+ROM_START( wyse286 )
+	ROM_REGION(0x20000, "bios", 0)
+	ROMX_LOAD( "wyse_tech_rev.a_250325-02_27128-15.bin", 0x18000, 0x4000, CRC(010f1c4d) SHA1(712d6ca4e4bdbc6b105c8691d612407edcfd9cf7), ROM_SKIP(1))
+	ROMX_LOAD( "wyse_tech_rev.a_250326-02_27128-15.bin", 0x18001, 0x4000, CRC(37fcd62b) SHA1(ada0e232387c8ba7067168f50f8b7a89eb824c44), ROM_SKIP(1))
+ROM_END
+
+// ***** Motherboards using the Acer (ALi) M1207 chipset
+
+// CMP enterprise CO.LTD. Phoenix 80286 ROM BIOS Version 3.00
+// ROM_SYSTEM_BIOS(26, "cmpa286", "CMP A286") /* (Chipset Acer (ALi) M1207-12) (BIOS release date:: 01-09-1986) (ISA8: 2, ISA16: 6) */
+ROM_START ( cmpa286 )
+	ROM_REGION(0x20000, "bios", 0)
+	ROMX_LOAD( "286-a286-even_16k.bin", 0x18000, 0x4000, CRC(30809487) SHA1(413de43ca7e1930cdf3c006718d8baf743a9ff1e), ROM_SKIP(1) )
+	ROMX_LOAD( "286-a286-odd_16k.bin", 0x18001, 0x4000, CRC(3a11aacf) SHA1(23185531ae10912b974048d3607b563e55d3fa96), ROM_SKIP(1) )
+ROM_END
+
+// AUVA VIP-M21502A BAM16-A0 - BIOS-String: DAR2-1105-061390-K0 - 286-BIOS AMI for AUVA 286, 02/08/1991 - ISA8:2, ISA16:5
+ROM_START( bam16a0 )
+	ROM_REGION(0x20000, "bios", 0)
+	ROMX_LOAD( "286-vip-m21502a-lo_32k.bin", 0x10000, 0x8000, CRC(413692db) SHA1(54bf664526b137cabf974c1fc659493e76243a88), ROM_SKIP(1) )
+	ROMX_LOAD( "286-vip-m21502a-hi_32k.bin", 0x10001, 0x8000, CRC(5db9db04) SHA1(8085384b943454a708be3104b47f6793d0040ab1), ROM_SKIP(1) )
+ROM_END
+
+
+
+// ***** Motherboards using the Chips SCAT 82C235 chipset
+
+// Biostar MB-1212C - ISA8:2, ISA16:5
+ROM_START ( mb1212c )
+	// 0: BIOS-String: ESC2-1223-083090-K2 - 286 BIOS AMI for MB-1212C version 1.1
+	ROM_REGION(0x20000, "bios", 0)
+	ROM_SYSTEM_BIOS(0, "mb1212_1", "mb1212_1")
+	ROMX_LOAD( "biostar mb-1212c.bin", 0x10000, 0x10000, CRC(153a783a) SHA1(acad4a3ffe93d3884dcb743c32d6317a132cda7b), ROM_BIOS(0) )
+	// 1: CHIPS SCAT BIOS Version 125D - MB-1212C
+	ROM_SYSTEM_BIOS(1, "mb1212_2", "mb1212_2")
+	ROMX_LOAD( "mb-1212c.bin", 0x10000, 0x10000, CRC(4675530a) SHA1(c34b1c67ac29695e565363f484e17ab5f8ddaad5), ROM_BIOS(1) )
+ROM_END
+
+
+// ***** Motherboards using the original Chips chipset: P82C202, P82C201, P82A203, P82A204, P82A205
+
+ROM_START( chips0 )
```

Comment:
CS8220 is the name Chips & Technologies used for their first-generation 82C201-82A205 CHIPset. I'm not sure it makes sense to separate these out, given that these five chips integrate almost no I/O registers whatsoever. (That would happen in the 82C206 Integrated Peripherals Controller, which was used with both the first-generation and "New Enhanced AT" CHIPsets.)